src/Nethermind/Nethermind.Serialization.Rlp/TxDecoder.cs

```diff
@@ -152,6 +187,29 @@ private void DecodeEip1559PayloadWithoutSig(T transaction, RlpStream rlpStream,
             transaction.AccessList = _accessListDecoder.Decode(rlpStream, rlpBehaviors);
         }
 
+        private void DecodeShardBlobPayloadWithoutSig(T transaction, RlpStream rlpStream, RlpBehaviors rlpBehaviors)
+        {
+            transaction.ChainId = rlpStream.DecodeULong(allowLeadingZeroBytes: false);
+            transaction.Nonce = rlpStream.DecodeUInt256(allowLeadingZeroBytes: false);
+            transaction.GasPrice = rlpStream.DecodeUInt256(allowLeadingZeroBytes: false); // gas premium
+            transaction.DecodedMaxFeePerGas = rlpStream.DecodeUInt256(allowLeadingZeroBytes: false);
```

Comment:
I put breakpoint here and I ran TxDecoder tests - looks like it wasn't covered by tests